docs/docs/code/nodejs/auth/README.md

```diff
@@ -20,7 +20,7 @@ export default defineComponent({
     }
   },
   async run({ steps, $ }) {
-    const web = new WebClient(this.slack.$auths.oauth_access_token)
```

Comment:
Changing references from `$auths` -> `$auth`. Data is exposed in the singular object

Reply:
Thanks, good catch